zest/releaser/pypi.py

```diff
@@ -176,26 +187,41 @@ def reload(self):
         upload fails, you edit the .pypirc file to fix the account
         settings, and tell release to retry the command.
         """
-        self._read_configfile(use_setup_cfg=self.use_setup_cfg)
+        self._read_configfile()
+    
+    def zest_releaser_config(self):
```

Comment:
This is exactly the same code as above. A utility function (or inheritance :-) ) should solve that.

Reply:
done